src/TypeSpec.Extension/Emitter.Csharp/src/lib/clientModelBuilder.ts

```diff
@@ -302,6 +302,7 @@ function setUsage(
     models: Map<string, InputModelType | InputEnumType>
 ) {
     for (const [name, m] of models) {
+        if (m.Usage !== undefined && m.Usage !== Usage.None) continue;
```

Comment:
Why are some models already set  `Usage`?

Reply:
These are the overridden ones by `@usage`.